README.md

```diff
@@ -1,3 +1,4 @@
+[![Coverage Status](https://coveralls.isgtesting.com/repos/github/sameer2209-arm/devicedb/badge.png)](https://coveralls.isgtesting.com/github/sameer2209-arm/devicedb)
```

Comment:
This is internal link. Don't put this in public repository because it is not publicly available

Reply:
But this is embedded link which adds coverage badge to the repo. It will have to be added to the readme if we want coverage badge

Reply:
@sameer2209-arm Could we collect these padges and links in edge-ci repository readme. It could be nice place to collect information

Reply:
@OPpuolitaival According to the coveralls docs, it is supposed to work this way. I think what you are suggesting might not be feasible here.

Reply:
Sould we change to public coveralls in public repositories?

Reply:
Accepted by @marhil01 that we can use public https://coveralls.io/ for public repositories. And it is enough to have results only in one coveralls. So no need to push also for private one.

Reply:
@OPpuolitaival What about the private repos ?

Reply:
@OPpuolitaival We would have to maintain two different coveralls for prvt and public repo. We won't have a single consolidated view. What is the risk with this approach ?

Reply:
@sameer2209-arm : Can we encrypt the internal urls on these public repos or check for any such plugins ? If not, please remove the embedded code for badges for now.

Reply:
@mohitsingalarm I don't think we can encrypt these URLs as they are embedded code generated by coveralls. And neither can we move these to edge-ci as according to coveralls docs these URLs need to be added to the readme of the repo to display the badge there.

Reply:
I am removing the badge URLs as of now as they have some rendering issue and also this issue of exposing internal links. We will take this up later in a different PR.